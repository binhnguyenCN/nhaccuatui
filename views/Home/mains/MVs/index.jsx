// libs
import React from 'react';
// layouts
import CategoryTitle from '../../components/CategoryTitle';
import MVList from './MVList';
// others
import CategoryTitles from '@/data_source/CategoryTitle';
import styles from './styles.module.scss';

const MVs = ({ videoList }) => (
  <div className={styles['mv-wrapper']}>
    <div className={styles['mv-inner']}>
      <CategoryTitle title={CategoryTitles.mvs} />
      <MVList MVs={videoList} />
    </div>
  </div>
);

export default MVs;
