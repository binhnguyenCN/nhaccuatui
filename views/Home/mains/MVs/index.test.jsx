// libs
import renderer from 'react-test-renderer';
// layouts
import MVs from '.';
// mocks
import mockData from '@/mocks/mockApi';

describe('render MVs', () => {
  it('should renders correctly', () => {
    // act
    const tree = renderer
      .create(<MVs videoList={mockData.videoList} />)
      .toJSON();
    // assert
    expect(tree).toMatchSnapshot();
  });
});
