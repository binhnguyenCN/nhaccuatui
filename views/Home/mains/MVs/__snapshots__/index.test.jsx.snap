// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MVs should renders correctly 1`] = `
<div
  className="mv-wrapper"
>
  <div
    className="mv-inner"
  >
    <h3
      className="category-title"
    >
      <span
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <span>
          mv hot
        </span>
        <span>
          <span
            aria-label="right"
            className="anticon anticon-right"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </span>
      </span>
    </h3>
    <ul
      className="mv-list"
    >
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={230}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
              title="Anh Lo Cho Em Hết"
              width={410}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:48
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Anh Lo Cho Em Hết
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2022/04/19/b/5/c/c/1650363937009_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Đàm Vĩnh Hưng
            </a>
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2021/03/02/2/8/6/4/1614651512512_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              , Ricky Star
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/20/f/d/f/c/1650435244252_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={230}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/20/f/d/f/c/1650435244252_640.jpg"
              title="Quan Tâm Em Mỗi Ngày"
              width={410}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:05
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/20/f/d/f/c/1650435244252_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Quan Tâm Em Mỗi Ngày
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2019/07/22/f/e/a/2/1563760304939_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Hoàng Tôn
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/14/5/9/1/f/1649925760384_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/14/5/9/1/f/1649925760384_640.jpg"
              title="Tình Chết Trong Hững Hờ"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:34
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/14/5/9/1/f/1649925760384_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Tình Chết Trong Hững
            ...
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2019/09/17/7/2/a/2/1568707006619_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Lệ Quyên
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/19/3/c/c/7/1650344734007_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/19/3/c/c/7/1650344734007_640.jpg"
              title="Kết Thúc Như Bao Người"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:30
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/19/3/c/c/7/1650344734007_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Kết Thúc Như Bao Ngư
            ...
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2019/11/20/d/f/9/a/1574242685613_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Tăng Phúc
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/03/24/4/b/3/6/1648109917566_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/03/24/4/b/3/6/1648109917566_640.jpg"
              title="Chống Lại Thế Giới"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              03:39
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/03/24/4/b/3/6/1648109917566_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Chống Lại Thế Giới
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2020/12/15/e/f/5/9/1608025280378_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Ngô Kiến Huy
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650017639222_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650017639222_640.jpg"
              title="Có Em Là Đủ"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              03:33
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650017639222_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Có Em Là Đủ
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2017/12/14/b/1/0/7/1513223438750_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Will
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109700967_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109700967_640.jpg"
              title="Psychofreak"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              03:26
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109700967_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Psychofreak
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2022/04/05/a/d/b/b/1649140280761_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Camila Cabello
            </a>
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/0"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              , Willow
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109588068_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109588068_640.jpg"
              title="Bam Bam"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:16
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/16/8/e/a/0/1650109588068_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Bam Bam
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2022/04/05/a/d/b/b/1649140280761_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Camila Cabello
            </a>
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2021/06/25/5/6/f/6/1624608342223_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              , Ed Sheeran
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650032406914_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650032406914_640.jpg"
              title=" Vai \\"Ác\\""
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              05:44
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/15/9/4/a/6/1650032406914_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
             Vai "Ác"
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2020/09/07/9/8/7/e/1599446170457_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Phạm Quỳnh Anh
            </a>
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              , Linh Đan
            </a>
          </span>
        </div>
      </li>
      <li
        className="mv-item"
      >
        <div
          className="mv-thumbnail"
        >
          <span
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="topic"
              height={109}
              src="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
              title="Anh Lo Cho Em Hết"
              width={195}
            />
          </span>
          <div
            className="thumbnail-dark-layer"
          />
          <div
            className="thumbnail-play-icon"
          >
            <span
              aria-label="play-circle"
              className="anticon anticon-play-circle"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="play-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
                <path
                  d="M719.4 499.1l-296.1-215A15.9 15.9 0 00398 297v430c0 13.1 14.8 20.5 25.3 12.9l296.1-215a15.9 15.9 0 000-25.8zm-257.6 134V390.9L628.5 512 461.8 633.1z"
                />
              </svg>
            </span>
          </div>
          <div
            className="thumbnail-durations"
          >
            <span
              aria-label="customer-service"
              className="anticon anticon-customer-service"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="customer-service"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 128c-212.1 0-384 171.9-384 384v360c0 13.3 10.7 24 24 24h184c35.3 0 64-28.7 64-64V624c0-35.3-28.7-64-64-64H200v-48c0-172.3 139.7-312 312-312s312 139.7 312 312v48H688c-35.3 0-64 28.7-64 64v208c0 35.3 28.7 64 64 64h184c13.3 0 24-10.7 24-24V512c0-212.1-171.9-384-384-384zM328 632v192H200V632h128zm496 192H696V632h128v192z"
                />
              </svg>
            </span>
            <span>
              04:48
            </span>
          </div>
        </div>
        <div
          className="mv-info"
        >
          <span
            className="mv-title"
            href="https://avatar-ex-swe.nixcdn.com/mv/2022/04/22/d/8/1/d/1650617244595_640.jpg"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Anh Lo Cho Em Hết
          </span>
          <span
            className="artist-list"
          >
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2022/04/19/b/5/c/c/1650363937009_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Đàm Vĩnh Hưng
            </a>
            <a
              href="https://avatar-ex-swe.nixcdn.com/singer/avatar/2021/03/02/2/8/6/4/1614651512512_300.jpg"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              , Ricky Star
            </a>
          </span>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
